src/attr/_make.py

```diff
@@ -125,10 +125,16 @@ def attr(default=NOTHING, validator=None,
         value is converted before being passed to the validator, if any.
     :param metadata: An arbitrary mapping, to be used by third-party
         components.  See :ref:`extending_metadata`.
+    :param type: The type of the attribute.  In Python 3.6 or greater, the
+        preferred method to specify the type is using a variable annotation
+        (see PEP-526 ).  This argument is provided for backward compatibility.
```

Comment:
PEPs are written without dashes and maybe this should point to some docs?  in any case there shouldn't be a blank before the paren :)